cmd/pke/app/phases/runtime/kubernetes/kubelet_kernel_params.yaml.tmpl

```diff
@@ -1,3 +1,6 @@
 vm.overcommit_memory=1
+# vm.overcommit_ratio=100
```

Comment:
this would be only effective if overcommit_memory is 2 (with value 1, the overcommit_ratio is effectively infinite). why do we keep it here?